database-mysql/schema.sql

```diff
@@ -0,0 +1,14 @@
+DROP DATABASE IF EXISTS calendar;
+
+CREATE DATABASE calendar;
+
+USE calendar;
+
+-- Table will hold dates that a specific place is unavailable
+-- and the minimum number of days someone has to stay to reserve the location
+CREATE TABLE availabilityDays(
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    startDate DATE,
```

Comment:
Change to snake_case for column names. SQL is case insensitive.

Reply:
changed!